File: web/ng-clickhouse-client/src/app/pricing/mass-quotes.ts
export class MassQuote {
    symbol: String;
    settleDate: Date;
    marketBid: number;
    marketAsk: number;
    
    exchangeId: number;
    customerId: String;
    sessionId: String;
    pricingProfileId: number;
    instrumentType: String;
    deliverable: boolean;
    spotDate: String;
    bid: number;
    ask: number;
    tfxLarge: boolean;
    spread: number;
}

File: src/main/java/com/ssk/ng/clickhouseclient/ws/MassQuotes.java
package com.ssk.ng.clickhouseclient.ws;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class MassQuotes {

    private String symbol;
    private Date tradeDate;
    private Date spotDate;
    private Date settleDate;
    private int exchangeId;
    private String customerId;
    private String sessionId;
    private String stream;
    private double bid;
    private double ask;
    private double spread;
}
